pages/tools/argent-x.mdx

```diff
@@ -0,0 +1,62 @@
+
+## Installing Argent X 
+Follow the official Argent X installation instructions: https://www.argent.xyz/learn/how-to-create-an-argent-x-wallet/ 
+
+## Use Argent X in Madara
+Argent X includes the Mainnet, Sepolia, and Goerli networks by default, but establishing interaction with your local Madara chain requires manual configuration. This involves adding a custom network within Argent X's settings.
+
+### Configuring Argent X for Madara
+1. Open the Argent X wallet and navigate to Settings.
+![stack](../../static/argentX/step1.png)
+2. Select "Developer settings" and then "Manage networks".
+![stack](../../static/argentX/step2.png)
+3. Click the "Add network button".
+4. Fill in the following fields:
+    - Network Name: A friendly name for the Madara network.
+    - Chain ID:
+        To retrieve the Madara chain ID:<br></br>
+            1. Open the `madara/blob/main/crates/runtime/src/pallets.rs` file and locate the ChainId variable.
+             The placeholder value will be `SN_GOERLI`.
+
+            2. Execute the following POST request using a tool like curl to retrieve the chain ID from your remote Madara node (replace http://localhost:9944 with the appropriate remote endpoint URL).           
+                ``` bash
+                    curl --location 'http://localhost:9944' \
+                    --header 'Content-Type: application/json' \
+                    --data '{
+                        "id": 0,
+                        "jsonrpc": "2.0",
+                        "method": "starknet_chainId",
+                        "params": {}
+                    }'
+                ```
+                The response will be a JSON object containing the chain ID in the "result" field. For example: 
+                ``` bash
+                    {
+                        "jsonrpc": "2.0",
+                        "result": "0x534e5f474f45524c49",
+                        "id": 0
+                    }
+                ```
+                - Use an online hex converter tool (e.g., https://stark-utils.vercel.app/converter) to convert the provided hexadecimal value (0x534e5f474f45524c49 in this example) to a readable string representation.
+                - Enter the converted string value in the "Chain ID" field within the Argent X network configuration.
+    - RPC URL: `http://localhost:9944`
+    - Sequencer URL: `http://localhost:9944`
+
+![stack](../../static/argentX/step3.png)
+5. Save the new network configuration.
+
+Once you have added Madara as a network, you can now connect to it however your Starknet wallet still needs to be deployed before its ready to be used.
+
+### Deploying your Starknet wallet
+While creating an Argent X wallet generates a Starknet address, it remains undeployed until an initial transaction triggers its activation. This means the address can passively receive funds even prior to activation, but outgoing transactions and full functionality remain unavailable.
```

Comment:
`outgoing transactions and full functionality remain unavailable` this gives the image that you can't use Argent. I know you mention it in the next paragraph. Maybe we should just write, Don't worry, Argent handles this under the hood?